meorg_client/cli.py

```diff
@@ -124,6 +124,19 @@ def file_upload(file_path):
         click.echo(f.get("file"))
 
 
+@click.command("upload_parallel")
+@click.argument("file_paths", nargs=-1)
+@click.option(
+    "-n", default=2, help="Number of simultaneous parallel uploads (default=2)."
+)
+def file_upload_parallel(file_paths, n=2):
+    client = _get_client()
+    responses = _call(client.upload_files_parallel, files=list(file_paths), n=n)
```

Comment:
Isn't `file_paths` already a list (it'd be great if arguments also had typings

Reply:
It should have typings, will fix.

Reply:
Oh, the argument in `_call` requires that because the `click` library interprets iterables as a tuple. This just casts that to a list so that it is interpreted correctly by the client function.